test/test.wpcom.site.media.js

```diff
@@ -26,124 +26,103 @@ describe('wpcom.site.media', function(){
   // Create a testing_media before to start tests
 
   var testing_media;
-  before(function(done){
-    site.addMediaFiles(fixture.media.files[1], function(err, data) {
-      if (err) throw err;
-
-      testing_media = data ? data.media[0] : {};
-      done();
-    });
+  before( done => {
+    site.addMediaFiles(fixture.media.files[1])
+      .then( data => {
+        testing_media = data ? data.media[0] : {};
+        done();
+      })
+      .catch(done);
   });
 
-  after(function(done){
+  after( done => {
     // clean media added through of array by urls
-    site.deleteMedia(add_urls_array.media[0].ID, function(err, data) {
-      if (err) throw err;
-
-      site.deleteMedia(add_urls_array.media[1].ID, function(err, data) {
-        if (err) throw err;
-
-        site.deleteMedia(add_urls_array.media[2].ID, function(err, data) {
-          if (err) throw err;
-
-          site.deleteMedia(add_urls_object.media[0].ID, function(err, data) {
-            if (err) throw err;
-
-            done();
-          });
-        });
-      });
-    });
+    site.deleteMedia(add_urls_array.media[0].ID)
+      .then( () => site.deleteMedia(add_urls_array.media[1].ID) )
```

Comment:
Oh yeah. At first I was following the style, but then I figured that if we are going to import the Calypso style soon, I wouldn't worry about it since it will all change.

Do you prefer that I follow the current format here before we merge this?